x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/EsqlFunctionRegistry.java

```diff
@@ -259,6 +259,15 @@ public boolean functionExists(String functionName) {
         return defs.containsKey(functionName);
     }
 
+    public String functionName(Class<? extends Function> clazz) {
```

Comment:
This call is going to be used a lot so please add a reverse map (Class to FunctionDefinition).

Reply:
Reverse map added.